gradio/blocks.py

```diff
@@ -671,6 +671,11 @@ def postprocess_data(self, fn_index, predictions, state):
                         )
                     output_index = dependency["outputs"].index(component._id)
                     reordered_predictions[output_index] = value
+                # Unwrap list if only one output component to prevent
+                # wrapping the list into a tuple in the if len(reordered_predictions) == 1
+                # check
+                if len(reordered_predictions) == 1:
+                    reordered_predictions = reordered_predictions[0]
```

Comment:
We've used this utils function in many other places in the code
```suggestion
                    reordered_predictions = utils.resolve_singleton(reordered_predictions)
```